.github/workflows/pre-commit.yaml

```diff
@@ -1,5 +1,7 @@
 on:
-    pull_request:
+  pull_request:
+    branches:
+      - master
```

Comment:
Will the other branches like issueYYY etc... also have the pre-commit tests?

Reply:
If I read [the documentatio](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches) correctly, then yes.

The list of ``branches`` are those **targeted** by a PR, not those that a PR originates from.

Reply:
Indeed!

Reply:
From the documentation, it seems to me they won't. But there's a `branches-ignore` [option](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore), so you can do 
```
on:
  pull_request:
    branches-ignore:
      - release
```
instead.

Reply:
Oh, I merged too fast in this case :(
Let's see with the next PR and if it doesn't work I will implement your fix @MJKirk